File: ios/Sources/RisemaxiGraft/Classes/Options/DownloadBundleOptions.swift
import Foundation
import Capacitor

@objc public class DownloadBundleOptions: NSObject {
    private let bundleId: String
    private let checksum: String
    private let url: String

    init(bundleId: String, checksum: String, url: String) {
        self.bundleId = bundleId
        self.checksum = checksum
        self.url = url
    }

    func getBundleId() -> String {
        return bundleId
    }

    func getChecksum() -> String {
        return checksum
    }

    func getUrl() -> String {
        return url
    }
}

File: android/src/main/java/com/risemaxi/graft/classes/options/DownloadBundleOptions.java
package com.risemaxi.graft.classes.options;

import androidx.annotation.NonNull;

public class DownloadBundleOptions {

    @NonNull
    private final String bundleId;

    @NonNull
    private final String checksum;

    @NonNull
    private final String url;

    public DownloadBundleOptions(@NonNull String bundleId, @NonNull String checksum, @NonNull String url) {
        this.bundleId = bundleId;
        this.checksum = checksum;
        this.url = url;
    }

    @NonNull
    public String getBundleId() {
        return bundleId;
    }

    @NonNull
    public String getChecksum() {
        return checksum;
    }

    @NonNull
    public String getUrl() {
        return url;
    }
}
